Library/Homebrew/download_strategy.rb

```diff
@@ -1368,6 +1389,7 @@ def self.detect_from_symbol(symbol)
     when :bzr                    then BazaarDownloadStrategy
     when :svn                    then SubversionDownloadStrategy
     when :curl                   then CurlDownloadStrategy
+    when :brewed_curl            then BrewedCurlDownloadStrategy
```

Comment:
```suggestion
    when :homebrew_curl            then HomebrewCurlDownloadStrategy
```

Thoughts on this naming, @Homebrew/maintainers? "brewed" is something we only real use in e.g. `HOMEBREW_FORCE_BREWED_CURL`.

Reply:
`:homebrew_curl` is fine too. It's probably a bit clearer for users who are less familiar with Homebrew.